### ISSUE TITLE
Missing Admin Colour Scheme Styles

### ISSUE THREAD

**Issue by tomjn** [humanmade/altis-cms#114]
#4667de on white fails the WCAG AAA contrast guidelines for normal text when ran through a contrast checker for. #2448CC was the closest that passed all tests when just adjusting darkness.

Some gutenberg fields are Altis blue, some aren't, e.g. the font size in block settings is blue, but the numerical input is not:

<img width="273" alt="Screenshot 2019-10-11 at 16 33 27" src="https://user-images.githubusercontent.com/58855/66664678-e1b24a00-ec44-11e9-8c32-fa2ed0684bc4.png">

Gutenberg tab highlights are in WP blue not Altis blue:

<img width="308" alt="Screenshot 2019-10-11 at 16 33 01" src="https://user-images.githubusercontent.com/58855/66664646-d2330100-ec44-11e9-9bdb-9420d95c458b.png">

**Comment by tomjn** [comment]
The help tab still uses the original blue which has much more cyan hue in it than the Altis colours, so it clashes with the main scheme:

<img width="495" alt="Screenshot 2019-10-11 at 17 00 18" src="https://user-images.githubusercontent.com/58855/66666527-b29dd780-ec48-11e9-9cff-4c0474d34e71.png">

**Comment by grarighe** [comment]
We're not going for WCAG AAA as far as I'm aware. `#4667DE` passes for WCAG AA.

> Some gutenberg fields are Altis blue, some aren't, e.g. the font size in block settings is blue, but the numerical input is not.

Well spotted. This should be addressed. Also the tabs highlights should be `#4667DE`.

> The help tab still uses the original blue which has much more cyan hue in it than the Altis colours, so it clashes with the main scheme

I guess the colour scheme for the help tab should match the same as the docs. But again, even in the docs, in the menu the tab highlight should be `#4667DE` and the correct colour for the background is `off-white` `#f3f5f9`. This colour can also be used for the icons in the admin menu (which should be white on hover). 

![Screenshot 2019-11-29 at 12.20.38.png](https://images.zenhubusercontent.com/58355f6bb9dc3c622022657a/9557ca92-5552-438e-bae8-8aba125d99ba)

**Comment by rmccue** [comment]
> I guess the colour scheme for the help tab should match the same as the docs.

Ah, I don't think we ever updated this since I stole the CSS from EK :)

**Comment by kevinlangleyjr** [comment]
I've opened two PR's for this issue:

Documentation styles: https://github.com/humanmade/altis-documentation/pull/110
CMS styles: https://github.com/humanmade/altis-cms/pull/174

The only thing I did not address in this PR is the gutenberg fields shown in the first screenshot within the issue. I did not address that as I am not seeing the issue any longer.

![Screen Shot 2020-01-09 at 2.11.18 PM.png](https://images.zenhubusercontent.com/544ec2c47c6ccc6277a58b71/0a5c295f-47c1-436a-a3fc-bf5eb5b0255c)
![Screen Shot 2020-01-09 at 2.11.26 PM.png](https://images.zenhubusercontent.com/544ec2c47c6ccc6277a58b71/b70f568e-e919-4053-8b46-2db81e7f7756)

**Comment by roborourke** [comment]
Thanks @kevinlangleyjr !
